app/views/playlists.js

```diff
@@ -0,0 +1,80 @@
+import Mn from 'backbone.marionette';
+import PlaylistView from './playlist';
+import application from '../application';
+import Playlist from '../models/playlist';
+
+
+const Playlists = Mn.CompositeView.extend({
+
+    template: require('./templates/playlists'),
+
+    childViewContainer: '#playlist-list',
+
+    childView: PlaylistView,
+
+    ui: {
+        addPlaylist: '#add-playlist',
+        playlistText: '#playlist-text',
+    },
+
+    events: {
+        'click p': 'changePlaylist',
+        'click #add-playlist': 'createPlaylist',
+        'focusout #add-playlist': 'focusoutAddPlaylist',
+        'keyup #playlist-text': 'keyupPlaylistText'
+    },
+
+    initialize() {
+        this.listenTo(this.collection, 'remove', this.removedPlaylist);
+    },
+
+    keyupPlaylistText(e) {
+        let ui = this.ui;
+        if(e.keyCode == 13) {
+            let newPlaylist = new Playlist();
+            newPlaylist.set('title', ui.playlistText.val());
+            application.allPlaylists.add(newPlaylist);
+            ui.playlistText.val('');
+            this.focusoutAddPlaylist();
+        }
+    },
+
+    removedPlaylist() {
+        $("#all-song").addClass('selected');
+    },
+
+    focusoutAddPlaylist() {
+        let ui = this.ui;
+        if (ui.playlistText.val() == '') {
+            ui.addPlaylist
+                    .addClass('add-playlist')
+                    .removeClass('input');
+        }
+    },
+
+    changePlaylist(e) {
+        let playlist = $(e.currentTarget);
+        let playlists = $(".playlists p");
```

Comment:
I think you're matching all _playlists_ items in the current view. In this case, prefer the use of `this.$('p')`, which is more performant as it internally does a `this.$el.find('p')` instead of relying to the global jQuery instance.

Consider the global jQuery as a call out-f-the-scope of the view, which should never occurs, else you break the concerns of the view (it tries to manipulate outside elements), which introduces side-effects complex to debug.